collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -279,7 +279,7 @@ docker-rancher() {
 
   for RANCHERAGENT in $RANCHERAGENTS; do
     docker inspect $RANCHERAGENT > $TMPDIR/rancher/containerinspect/agent-$RANCHERAGENT 2>&1
-    docker logs $SINCE_FLAG -t $RANCHERAGENT > $TMPDIR/rancher/containerlogs/agent-$RANCHERAGENT 2>&1
+    docker logs $SINCE_FLAG -t $RANCHERAGENT | sed 's/token.*/token REDACTED/g' > $TMPDIR/rancher/containerlogs/agent-$RANCHERAGENT 2>&1
```

Comment:
The pipe will pass stdout by default, combining the streams before the pipe should ensure both get to `sed` and the log file.

The shorthand `|&` could also be used, but for consistency `2>&1`.

```suggestion
    docker logs $SINCE_FLAG -t $RANCHERAGENT 2>&1 | sed 's/token.*/token REDACTED/g' > $TMPDIR/rancher/containerlogs/agent-$RANCHERAGENT 2>&1
```